common/src/main/java/org/apache/rocketmq/common/MixAll.java

```diff
@@ -339,6 +339,8 @@ public static void properties2Object(final Properties p, final Object object) {
                     String key = first.toLowerCase() + tmp;
                     String property = p.getProperty(key);
                     if (property != null) {
+                        // remove spaces before and after
```

Comment:
what about we also trim the method `public static String properties2String(final Properties properties)` ?